tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-product-mapper-test.php

```
@@ -927,4 +927,101 @@ public function test_delete_field_always_returns_null_for_active_products() {
+		$product->save();
+
+		$callback = static fn() => false;
+		add_filter( 'wc_stripe_agentic_commerce_should_sync_product', $callback );
+
+		try {
+			$this->assertFalse( \WC_Stripe_Agentic_Commerce_Product_Mapper::should_sync_product( $product ) );
+		} finally {
+			remove_filter( 'wc_stripe_agentic_commerce_should_sync_product', $callback );
+			$product->delete( true );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method test_should_sync_product_respects_filter_returning_false is not named in camelCase. (undefined)

(CamelCaseMethodName)